quantgov/estimator/structures.py

```diff
@@ -85,3 +113,41 @@ class CandidateModel(
             parameter values to test as values
     """
     pass
+
+
+class QGLdaModel(BaseEstimator, TransformerMixin):
```

Comment:
I don't like either the prefix or the Model specifier. I'd call this GensimLDA or something like that.